for-each.js

```diff
@@ -27,21 +31,31 @@ Output:
 */
 
 export function makeReversedArrayOfTypes(arr) {
-    return [];
+    const Pets = [];
+        arr.forEach((item) => {
+            Pets.unshift(item.type);
+        });
+        return Pets;
 }
 
 /*
 Output:
 
-[
+
     { nombre: 'spot', tipo: 'dog' },
     { nombre: 'rover', tipo: 'dog' },
     { nombre: 'jumpy', tipo: 'frog' },
     { nombre: 'einstein', tipo: 'cat' },
-]
+
 */
 
 export function makeSpanishLanguageArray(arr) {
-    return [];
-}
+    const newArr = [];
+    arr.forEach((item) => {
+
+        newArr.push(`nombre: ${item.name}, tipo: ${item.type}`);
```

Comment:
you don't need the backticks or template literal here. because the value of each is already a string item.name will give you a string.